src/backends/scoop.rs

```diff
@@ -0,0 +1,95 @@
+use std::collections::{BTreeMap, BTreeSet};
+
+use color_eyre::Result;
+use serde::{Deserialize, Serialize};
+
+use crate::cmd::{run_command, run_command_for_stdout};
+use crate::prelude::*;
+
+#[derive(Debug, Copy, Clone, Default, PartialEq, Eq, PartialOrd, Ord, derive_more::Display)]
+pub struct Scoop;
+
+#[derive(Debug, Clone, Default, PartialEq, Eq, PartialOrd, Ord, Serialize, Deserialize)]
+#[serde(deny_unknown_fields)]
+pub struct ScoopGetOptions {}
+
+impl Backend for Scoop {
+    type Options = ScoopGetOptions;
+
+    fn map_required(
+        packages: BTreeMap<String, Package<Self::Options>>,
+        _: &Config,
+    ) -> Result<BTreeMap<String, Package<Self::Options>>> {
+        Ok(packages)
+    }
+
+    fn query(config: &Config) -> Result<BTreeMap<String, Self::Options>> {
+        if Self::version(config).is_err() {
+            return Ok(BTreeMap::new());
+        }
+
+        let output = run_command_for_stdout(
+            [
+                "scoop.cmd",
```

Comment:
Why `scoop.cmd` and not just `scoop`? I'm assuming just `scoop` doesn't work but it's odd we don't have do do that for the `winget` backend. I'll do some testing on windows.

Reply:
I'm not 100% sure here, but at least on my system it looks like scoop isn't installed as a binary (.exe) but is a powershell command and then there's a shim named scoop.cmd to call it from cmd (and other shells like nu). I initially tried with just scoop and it didn't work for me at least :)